modules/themoneytizerBidAdapter.md

```diff
@@ -0,0 +1,36 @@
+# Overview
+
+```
+Module Name: The Moneytizer Bid Adapter
+Module Type: Bidder Adapter
+Maintainer: tech@themoneytizer.com
+```
+
+## Description
+
+Module that connects to The Moneytizer demand sources
```

Comment:
could we include a table explaining all possible params that your adapter accepts: pid, test, placement, formats, isInternal, baseUrl

Reply:
- The 'placement, formats, and isInternal' were for development only and have been removed

- Table added with pid, test and baseUrl